Sources/SentryCrash/Recording/Tools/SentryCrashObjC.c

```diff
@@ -232,7 +232,7 @@ decodeIsaPointer(const void *const isaPointer)
 #if ISA_TAG_MASK
     uintptr_t isa = (uintptr_t)isaPointer;
     if (isa & ISA_TAG_MASK) {
-#    if defined(__arm64__)
```

Comment:
I found more usages of this approach: https://github.com/getsentry/sentry-cocoa/search?q=defined%28__arm64__%29&unscoped_q=defined%28__arm64__%29

Does it make sense to change it everywhere?

Reply:
I won’t pretend to be an expert on either Sentry or ARM architectures, but I believe that at least some of the other places where it’s used, are valid, in that they relate to things like page sizes, which Apple Silicon retains from the iOS CPUs.

Reply:
The build fails because `kCFCoreFoundationVersionNumber_iOS_8_x_Max` doesn't exist in macOS. It is located in CoreFoundation/CFBase.h of iOS. Before Apple Silicon, only iPhones and iPads used `arm64`. Now we can't use `arm64` to find out whether we are on an iPhone/iPad or a Mac. So we have to use `TARGET_OS_IOS`. The other usages of `__arm64__` pointed out by you @bruno-garcia are processor and not operating system specific so it is fine to keep them.